TextRPG_Team25/Battle.cs

```diff
@@ -40,7 +41,52 @@ private void PlayerAttack(Monster selected)
                 Console.WriteLine($"HP {currentHp}-> HP {selected.HP}");
             }
 
-            
+  
         }
+
+        private void SpawnEnemy() // 몬스터 소환
+        {
+            Random rand = new Random();
+            int spawnNum = rand.Next(1, 4);
+            RanMonsterSpawn(spawnNum);
+        }
+
+        private void RanMonsterSpawn(int spawnNum) // 랜덤 몬스터 소환
+        {
+            Random rand = new Random();
+            for (int i = 0; i < spawnNum; i++)
+            {
+                int num = rand.Next(0, 3);
+                monster.FieldMonster.Add(monster[num]);
+            }
+        }
+
+        public void StartBattle() // 배틀 시작
+        {
+            SpawnEnemy();
+            for(int i = 0; i < monster.FieldMonster.Count; i++)
+            {
+                Console.WriteLine($"Lv.{monster.FieldMonster[i].level}  {monster.FieldMonster[i].name}  HP  {monster.FieldMonster[i].hp}");
+            }
+
+            Console.WriteLine("");
+            Console.WriteLine("[내정보]");
+            Console.WriteLine($"Lv.{player.level}  {player.name} ({player.job})");
+            Console.WriteLine($"HP {player.maxHp}/{player.hp}\n");
+            Console.WriteLine("0. 취소\n");
+            Console.WriteLine("대상을 선택해주세요.\n");
+            Console.WriteLine(">>");
+        }
+
+        public void SelectTarget() // 타겟 몬스터 지정
+        {
+            string input = Console.ReadLine();
+            int num = int.Parse(input);
+            if(num > monster.FieldMonster.count || num < 0 || !monster.FieldMonster[num].isLive)
+            {
+                Console.WriteLine("잘못된 입력입니다.");
```

Comment:
예외처리 조건 좋아요! 몬스터 앞의 숫자가 플레이어에게 1, 2, 3으로 보일텐데 인덱스는 0부터기 때문에 input -1로 넣어주어야 한다는 점만 유의해주세여 (현재는 숫자표시 기능이 없습니다) 

그리고 이렇게 조건이 많을 때에는 else로 처리하는 쪽이 좀 더 깔끔합니당!
```suggestion
          if (input <= fieldMonsters.Count())
          {
              PlayerPhase(fieldMonsters[input - 1]);
          }
          else
          {
              Console.WriteLine("잘못된 입력입니다.");
          }
```

Reply:
죽은 몬스터가 선택될때도 "잘못된 입력입니다" 텍스트가 출력되어야 하기 떄문에 if문 조건식에 관련 조건이 추가되어야 할 것 같습니다.